composer.json

```diff
@@ -30,7 +30,9 @@
         "zendframework/zend-math": "^2.6",
         "zendframework/zend-servicemanager": "^2.7.5 || ^3.0.3",
         "zendframework/zend-session": "^2.8",
-        "zendframework/zend-uri": "^2.5"
+        "zendframework/zend-uri": "^2.5",
+        "psr/http-client": "^1.0",
```

Comment:
These should be near the other `psr/*` libs above

Reply:
@Ocramius I can do that… but have you seen my request in regards PSR-18 not being compatible with PHP 5.6? ➡️ https://github.com/zendframework/zend-validator/issues/259#issuecomment-482839690

Reply:
Yeah, I think we need to bump the dependency here: it is long overdue and should just be done.

Also, I'm using this occasion to re-express to @zendframework/community-review-team the fact that bumping PHP version can be freely done in minor releases.

Reply:
I raised https://github.com/zendframework/maintainers/issues/59 about this